ifrs17/Report/ReportStorage.ipynb

```diff
@@ -110,6 +110,15 @@
                 "\n    protected readonly IExportVariable export;",
                 "\n    private readonly Systemorph.Vertex.Hierarchies.IHierarchicalDimensionCache hierarchicalDimensionCache;",
                 "\n    private readonly Systemorph.Vertex.Pivot.Builder.Interfaces.IPivotFactory reportFactory;",
+                "\n    private int periodicityInMonth;",
+                "\n",
+                "\n    public ProjectionConfiguration[] OrderedProjectionConfigurations {get; private set;}",
+                "\n",
+                "\n    public async Task<ProjectionConfiguration[]> GetOrderedProjectionConfigurationsAsync() => await workspace.Query<ProjectionConfiguration>()",
```

Comment:
It's already here. But the GetOrderedProjectionConfigurationsAsync should react to changes of Args.Year and Args.Month. It is not a const.